### DISCUSSION TITLE
Connect to vpn from internet

### DISCUSSION THREAD

**Discussion by ithotline** [Ysurac/openmptcprouter#1974] [Q&A]
Hi.

How would one go about connecting in to your lan from the outside?

**Comment by Ysurac** [comment]
You set a VPN on VPS side, then you can do what you want with route and by opening access from VPN on the router in the firewall, or you can set a VPN on router and redirect port from VPS to it.

**Reply by ithotline** [reply]
Thanks for the response.

There is already alot of vpn going on between the router and vps - is it possible to use this or do i have to setup new vpn between router and vps?

The goal i want to reach is to connect to vps and get acces to lan behind router.

**Reply by Ysurac** [reply]
VPN can be used, you need to open allow it in input way on the router firewall. By default only ping is allowed.
LAN routes are added by default on the VPS.
